### ISSUE TITLE
The game needs one additional intermediate step/goal for the player to reach before Malone sends them to get a passport.

### ISSUE THREAD

**Issue by Nextej** [loiste-interactive/Obenseuer-Issues#848]
After more or less 20 hours of play time, I've done pretty much everything story-related (trying really hard no to push the story), so with no actual tasks left I had 10 finished renovations, awaiting 2 more, 3 moved in tenants with made space for 2 more and that's barely scratching the huge amount the renovations there are left... It feels like the Malone sends you too soon for the passport, kicking off all the story related content while there's so much renovations left.

1. At this stage, cut off the content for Rob-In Repair out of Samuel's list (Oskutin said there will be two more contractors, but currently only the highest levels are locked, lock away everything that's not for Samuel now).
2. It takes too long to get enough money for makeshift level renovations, and it takes too much to wait for them to finish, I've already made my proposed changes in another ticket #812 this should help the pacing so the amount of time player spends on renovations and the story content should be already more balanced.
3. But I think there's a need for an intermediate step/goal that will push story content a little further down the road. One more thing Malone sets up for player and sends them somewhere when done before sending them to get a passport. I think that after player starts 5 renovations, Malone should demand player to get 5 tenants settled in (that the most optimally would require at least 5 more renovations) and only after getting them in and talking to Malone he'll send the player to get the passport.
4. However it would be nice if Malone sent the player somewhere else too after starting those 5 renovations, I thought that it could be worth adding a new character and location somewhere, so the player can be taught about mashing, fermenting and distilling, so the Malone could send the player to that new character after 5 renovations is started and the passport is moved further the next step mentioned above.
5. There's one more issue, how soon the Radio Signal task becomes available in compare to how late the player gets OSRAT to complete optional objective in this task. I think it would be better to have it somehow split into two tasks, one part ends when player found the woman in bed and the body and informed Amanda about it (and umm, the police I guess), and the second part starts when Amanda wants to get to know more about the whole situation and player has OSRAT already. Perhaps lower Amanda's standards (swap her around with the security guard under the Greenhouse stairs) and require her first get an apartment to continue the task, locking the further dialogue behind something like "I'd like to get to know what's going on here, but first I need to get some actual place to live".

**Comment by Nextej** [comment]
Since I see the release deadline is tight, at least add that one additional step of "Rent apartments to 5 tenants" to the "Getting Started" task, before Malone gives "Getting a Passport" task.

You could also have a concurent objective to the "Start 5 renovations" being "Get the first tenant" alongside (unless player rented the apartment already before going back to Malone telling them about first renovation), so players have an introduction to the next objective, which would now be reduced to just "Rent apartments to 4 tenants". Also so it's not too derivative in numbers as the previous "Start 5 renovations".

**Comment by Nextej** [comment]
As it turns out, the game has "Find someone to rent apartment to" objective for "Getting Started" that appears after "Start 5 renovations" if the player hadn't had rented any apartment before, so would be nice if that objective was moved after "Start first renovation" objctive and was concurently available as the "Start 5 renovations" objective (similarly to the "Survive in Obenseuer" quest). And after finishing both "Start 5 renovations" and "Find someone to rent apartment to" (if the played hadn't rented before), have the "Rent apartments to 4 tenants" objective (virtually the same condition as the previous objective, just the number is now 4), which unlocks getting the passport questline.

**Comment by Nextej** [comment]
Also, Malone sending optionally player to Speakeasy could be moved after getting the first tenant (which now would be the after starting the first renovation). This way player would be held just a little longer off before overwhelming them with 1 new location, 3 new tasks and another new location (deekula B), while allowing them to get more familiar with how renovation and renting works, which I think is important to get player hooked on the concept before you start throwing in more quests or story related quests.

So the final proposed quest progression of getting started:
![image](https://github.com/loiste-interactive/Obenseuer-Issues/assets/44178400/15a8d4d8-7f8f-4e4f-9350-b38a63029ddd)

Red asterisk - "Start 5 more renovations" if player started any more renovations before returning to Malone than just the first one, add the additional ones already to the counter #879
Green Aastrisk - "Find someone to rent the apartment to" if player had rented the first one already before returning to Malone, skip this objective and add the next one already about visiting Speakeasy.

The whole thing would now show basics (farming, renovations and renting) -> having a time to start first renovations and get familiar with the place -> give some more quests at the pub to the player to open the world more -> prepare the tenement for more residents, player should have 1/3 of the tenement in livable makeshift conditions, perhaps some shabby ones, staircase or basement upgrade -> give some more quests, now story related

**Comment by Nextej** [comment]
This one is crucially important, also, Oskutin was talking about making Malone send you for an additional objective to do something with getting furniture/tools, so it would be good to leave an option of opportunity of expanding this quest after completing 5 more renovations.

**Comment by Zuntti** [comment]
Maybe the basement should also be examined first before it can be renovated, but it would be locked first. You could ask Malone for the keys and he would say that he would talk about it later when enough renovations have been done. He would give the keys when the five renovations have been done. And there would be something about the basement that Malone wouldn't want to show.

**Comment by Nextej** [comment]
Also, from my recent playthrought, it feels like the player should have electricity already in the apartment before they reach bazaar, but it would be too much fitting that into "Getting Started", so I thought that it would be good if Rob-In Repair required to have electricity in the tenement before player is allowed to renovate anything at them.

**Comment by Nextej** [comment]
I'm also a bit torn on Tenement Dirty Water being locked behing Tenement Shit Electricity, simplies solution would be to just separate them so they can be started concurrently, but I'm not sure about it, alternative is to cut down awaiting time for each upgrade, especially electricity. There's also that old suggestion about being able to speed up specific renovation with upfront payment #905, could also solve the issue.
